File: app/src/main/java/im/bernier/petfinder/activity/BaseActivity.java
package im.bernier.petfinder.activity;

import android.support.v7.app.AppCompatActivity;

/**
 * Created by Michael on 2017-03-24.
 * Breather Products Inc.
 */

public class BaseActivity extends AppCompatActivity {

    protected boolean isTablet() {
        return getResources().getConfiguration().smallestScreenWidthDp >= 600;
    }
}

File: app/src/main/java/im/bernier/petfinder/activity/BaseActivity.kt
package im.bernier.petfinder.activity

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity

/**
 * Created by Michael on 2017-03-24.
 * Breather Products Inc.
 */

@SuppressLint("Registered")
open class BaseActivity : AppCompatActivity() {

    protected val isTablet: Boolean
        get() = resources.configuration.smallestScreenWidthDp >= 600
}
